swig/python/pyproject.toml

```diff
@@ -0,0 +1,32 @@
+[build-system]
+requires = ["setuptools", "oldest-supported-numpy"]
```

Comment:
> Any reason not to just use a broad `numpy` dependency?

just copying what rasterio does...

Reply:
ok, I've changed that to ``numpy`` in an extra commit. I suspect rasterio does what it does to be able to do binary wheels using ancient toolchain so that they can work on the largest number of systems. Plain numpy should be fine in a GDAL context where we don't provide binary wheels